js/bitfinex2.js

```diff
@@ -1013,10 +1013,10 @@ module.exports = class bitfinex2 extends Exchange {
         for (let i = 0; i < orderbook.length; i++) {
             const order = orderbook[i];
             const price = this.safeNumber (order, priceIndex);
-            const signedAmount = this.safeNumber (order, 2);
-            const amount = Math.abs (signedAmount);
-            const side = (signedAmount > 0) ? 'bids' : 'asks';
-            result[side].push ([ price, amount ]);
+            const signedAmount = this.safeString (order, '2');
```

Comment:
You shouldn't convert 2 to '2'  here because that's an index inside the array. I will fix it this time but just a reminder for the future